OpenWeb-SampleApp/UI/AuthenticationPlayground/Views/AuthenticationPlaygroundVC.swift

```diff
@@ -185,9 +212,37 @@ private extension AuthenticationPlaygroundVC {
             make.trailing.equalTo(view.safeAreaLayoutGuide).offset(-Metrics.horizontalMargin)
         }
 
+        scrollView.addSubview(switchAuthenticationWithCustomFields)
```

Comment:
Just make it a `UIStackView`, it will be so much easier.

Reply:
Done